src/pd/pd.rs

```diff
@@ -241,6 +265,19 @@ impl<T: PdClient> Runner<T> {
         }
 
         stats.set_available(available);
+        stats.set_bytes_read(
+            self.store_stat.engine_total_bytes_read - self.store_stat.engine_last_total_bytes_read,
+        );
+        stats.set_keys_read(
+            self.store_stat.engine_total_keys_read - self.store_stat.engine_last_total_keys_read,
+        );
+        self.store_stat.engine_last_total_bytes_read = self.store_stat.engine_total_bytes_read;
+        self.store_stat.engine_last_total_keys_read = self.store_stat.engine_total_keys_read;
+
+        for peer in self.region_peers.values() {
```

Comment:
define a local metric and flush after the for loop like https://github.com/pingcap/tikv/blob/master/src/raftstore/store/store.rs#L1692

Reply:
I prefer handling write bytes/keys to PD worker too.